src/error.rs

```diff
@@ -107,8 +107,7 @@ impl Error {
             FileNotFound(_) => Self::FileNotFound(err_str),
             UnsupportedType => Self::NotSupported(err_str),
             ClaimVerification(_) | InvalidClaim(_) | JumbfParseError(_) => Self::Verify(err_str),
-            #[cfg(feature = "add_thumbnails")]
-            ImageError => Self::ImageError(err_str),
+            // ImageError => Self::ImageError(err_str),
```

Comment:
Is that one gone, then?

Reply:
I haven't added a "add_thumbnails" feature for the c2pa-c build - currently we always enable it, but I know Stein would prefer we always remove it. I'll clean this up.